src/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Tab.scss';
+import classnames from 'clsx';
+import {Typography} from '../Typography';
+
+export const tabSizes = ['small', 'default', 'big'];
+export const tabVariants = ['ghost'];
+export const tabColors = ['default'];
+
+export const Tab = ({label, onClick, size, isReversed, isDisabled, icon, variant, color, className, isSelected, ...props}) => {
+    let typoWeight = 'default';
+    if (size === 'small') {
+        typoWeight = 'light';
+    }
+
+    if (size === 'big') {
+        typoWeight = 'semiBold';
+    }
+
+    return (
+        <button
```

Comment:
Add the possibility to override the main element. See  @vindhya if you want help on this.

Reply:
what did you mean by override the main element?

I'll see with her :D 

Thanks